### PR TITLE
Added Data Structure courses in Hindi

### PR DESCRIPTION
**Pull request by anishhh238**
Added Data Structure courses in Hindi tht Helps students to learn DSA with professional, well-known Youtubers

## What does this PR do?
Add resource(s) | Remove resource(s) | Add info | Improve repo

### Description
This are the DSA course that helps students to learn DSA

### Why is this valuable (or not)?
Valuable for the students who wants to learn DSA

### How do we know it's really free?
It's totally free avilable on Youtube

### For book lists, is it a book? For course lists, is it a course? etc.
Yes, it is a course 

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [ ] Include author(s) and platform where appropriate.
- [ ] Put lists in alphabetical order, correct spacing.
- [ ] Add needed indications (PDF, access notes, under construction).
- [ ] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: courses/free-courses-hi.md [hunk 1]
```diff
@@ -185,6 +185,8 @@
 * [Java + DSA](https://www.youtube.com/playlist?list=PLfqMhTWNBTe3LtFWcvwpqTkUSlB32kJop) - Apna College (YouTube)
 * [Linked Lists - Level 1](https://www.youtube.com/playlist?list=PL-Jc9J83PIiF5VZmktfqW6WVU1pxBF6l_) - Pepcoding
 * [Stack Playlist \| Interview Questions \| Coding \| Tutorials \| Data Structures](https://www.youtube.com/playlist?list=PL_z_8CaSLPWdeOezg68SKkeLN4-T_jNHd) - Aditya Verma 
+* [Simplified DSA with FRAZ](https://www.youtube.com/playlist?list=PLKZaSt2df1gy75J3irj89a2vSGqeQdtDA)-FRAZ
```
Comment (shrn01) [inline courses/free-courses-hi.md:188]:
```suggestion
* [Simplified DSA with FRAZ](https://www.youtube.com/playlist?list=PLKZaSt2df1gy75J3irj89a2vSGqeQdtDA) - Fraz
```

File: courses/free-courses-hi.md [hunk 2]
```diff
@@ -185,6 +185,8 @@
...
+* [Complete C++ Placement DSA Course](https://www.youtube.com/playlist?list=PLDzeHZWIZsTryvtXdMr6rPh4IDexB5NIA)-CodeHelp- by Babbar
```
Comment (shrn01) [inline courses/free-courses-hi.md:189]:
```suggestion
* [Complete C++ Placement DSA Course](https://www.youtube.com/playlist?list=PLDzeHZWIZsTryvtXdMr6rPh4IDexB5NIA) - CodeHelp by Babbar
```

**Review by shrn01 — COMMENTED** [review]
Please sort the resources alphabetically

**Comment by anishhh238** [comment]
Okay, I'll do it.

**Pull request merged**